packages/battery/battery_platform_interface/lib/enums/batteryState.dart

```diff
@@ -0,0 +1,11 @@
+/// Indicates the current battery state.
```

Comment:
Please, don't use camelCase in the name of this file, this file should be: `battery_state.dart`

Reply:
I renamed it now.